src-gui/src/renderer/components/modal/swap/pages/in_progress/WaitingForXmrConfirmationsBeforeRedeemPage.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, DialogContentText } from "@material-ui/core";
+import { TauriSwapProgressEventContent } from "models/tauriModelExt";
+import MoneroTransactionInfoBox from "../../MoneroTransactionInfoBox";
+
+export default function WaitingForXmrConfirmationsBeforeRedeemPage({
+  xmr_lock_txid,
+  xmr_lock_tx_confirmations,
+  xmr_lock_tx_target_confirmations,
+}: TauriSwapProgressEventContent<"WaitingForXmrConfirmationsBeforeRedeem">) {
+  return (
+    <Box>
+      <DialogContentText>
+        We are waiting for the Monero lock transaction to receive enough confirmations before we can sweep them to your address.
```

Comment:
Maybe something like "Your swap was successful. Due to Monero blockchain restrictions, we need 8 more confirmations (~16 minutes) to send the Monero to your redeem address."